probe-rs/src/probe/jlink/mod.rs

```diff
@@ -337,6 +337,14 @@ impl DebugProbe for JLink {
         "J-Link"
     }
 
+    fn speed(&self) -> u32 {
+        unimplemented!()
+    }
+
+    fn set_speed(&mut self, _speed_khz: u32) -> Result<u32, DebugProbeError> {
+        unimplemented!()
```

Comment:
Maybe we should implement this right away so users don't have crashes and this actually seems trivial: https://docs.rs/jaylink/0.1.1/jaylink/struct.JayLink.html#method.set_speed